### ISSUE TITLE
Provider is incompatible with terraform version 0.12.0

### ISSUE THREAD

**Issue by annedroiid** [terraform-providers/terraform-provider-pagerduty#134]
The pagerduty provider doesn't work with the latest release of Terraform.

### Terraform Version
0.12.0

### Affected Resource(s)
- provider "pagerduty"

### Terraform Configuration Files
```hcl
variable "pagerduty_token" {
  description = "The PagerDuty api token"
}

provider "pagerduty" {
  token = "${var.pagerduty_token}"
}

# Service for the lambda
resource "pagerduty_service" "our-service" {
  name                    = "Service name"
  description             = "Service description"
  escalation_policy       = "PA8LWK9"
  alert_creation          = "create_alerts_and_incidents"

  incident_urgency_rule {
    type = "constant"
    urgency = "high"
  }
}

resource "pagerduty_service_integration" "grafana" {
  service = "${pagerduty_service.our-service.id}"
  name = "Grafana"
  type = "events_api_v2_inbound_integration"
}

```

### Expected Behavior
The service and grafana integration should have been created.

### Actual Behavior
The following error was received
```
Initializing provider plugins...
- Checking for available provider plugins...

No available provider "pagerduty" plugins are compatible with this Terraform version.

From time to time, new Terraform major releases can change the requirements for
plugins such that older plugins become incompatible.

Terraform checked all of the plugin versions matching the given constraint:
    (any version)

Unfortunately, none of the suitable versions are compatible with this version
of Terraform. If you have recently upgraded Terraform, it may be necessary to
move to a newer major release of this provider. Alternatively, if you are
attempting to upgrade the provider to a new major version you may need to
also upgrade Terraform to support the new version.

Consult the documentation for this provider for more information on
compatibility between provider versions and Terraform versions.


Error: no available version is compatible with this version of Terraform
```

### Steps to Reproduce
1. `terraform apply`

**Comment by kylegoch** [comment]
Looks like support has been merged: https://github.com/terraform-providers/terraform-provider-pagerduty/pull/126

But a release with support has not been cut: https://github.com/terraform-providers/terraform-provider-pagerduty/blob/master/CHANGELOG.md#130-unreleased

**Comment by gordonbondon** [comment]
This has been released in [v1.3.0](https://github.com/terraform-providers/terraform-provider-pagerduty/releases/tag/v1.3.0)

**Comment by heimweh** [comment]
Hi @annedroiid ,

closing this for now but feel free to reopen at anytime. 

Best,

Alexander
